packages/server/initSentry.ts

```diff
@@ -0,0 +1,23 @@
+import * as Integrations from '@sentry/integrations'
+import * as Sentry from '@sentry/node'
+
+declare global {
+  namespace NodeJS {
+    interface Global {
+      __rootdir__: string
+    }
+  }
+}
+
+const APP_VERSION = process.env.npm_package_version
+Sentry.init({
```

Comment:
worth noting sentry is no longer in our request middleware. it still wraps uncaught exceptions & calls that we send to it